api/libproject/environments.rb

```diff
@@ -54,7 +54,7 @@ def env_with_defaults(env, config)
     :tanagra_auth_disable_checks => "false",
     :tanagra_auth_bearer_token => "true",
     :tanagra_underlay_files => "aou/test/SC2023Q3R1/SC2023Q3R1.json,aou/test/SR2023Q3R1/SR2023Q3R1.json",
```

Comment:
I think this should be: `aouSR2023Q3R1_aoutest,aouSC2023Q3R1_aoutest` for version 0.0.276 following this:  https://github.com/DataBiosphere/tanagra/blob/9a29b0f43846408cb84b282032d0ab3526303e2b/service/local-dev/run_server.sh#L52-L52